build/packaging/linux/bin/kend

```diff
@@ -203,6 +198,100 @@ start() {
         OPTIONS="$OPTIONS --autorestart.enable"
     fi
 
+    if [[ ! -z $KES_DB_TYPE ]] && [[ $KES_DB_TYPE != "" ]]; then
+        OPTIONS="$OPTIONS --dbtype $KES_DB_TYPE"
+        if [[ ! -z $KES_DB_DYNAMO_TABLENAME ]] && [[ $KES_DB_DYNAMO_TABLENAME != "" ]]; then
+            OPTIONS="$OPTIONS --db.dynamo.tablename $KES_DB_DYNAMO_TABLENAME"
+        fi
+
+        if [[ ! -z $KES_DB_DYNAMO_SINGLE ]] && [[ $KES_DB_DYNAMO_SINGLE -eq 1 ]]; then
+            OPTIONS="$OPTIONS --db.single"
+        fi
+
+        if [[ ! -z $KES_DB_DYNAMO_READONLY ]] && [[ $KES_DB_DYNAMO_READONLY -eq 1 ]]; then
+            OPTIONS="$OPTIONS --db.dynamo.read-only"
+        fi
+    fi
+
+    if [[ ! -z $GCMODE ]] && [[ $GCMODE != "" ]]; then
+        OPTIONS="$OPTIONS --gcmode $GCMODE"
+    fi
+
+    if [[ ! -z $KES_STATE_TRIE_CACHE_LIMIT ]] && [[ $KES_STATE_TRIE_CACHE_LIMIT != "" ]]; then
+        OPTIONS="$OPTIONS --state.trie-cache-limit $KES_STATE_TRIE_CACHE_LIMIT"
+    fi
+
+    if [[ ! -z $KES_STATEDB_CACHE_TYPE ]] && [[ $KES_STATEDB_CACHE_TYPE != "" ]]; then
+        OPTIONS="$OPTIONS --statedb.cache.type $KES_STATEDB_CACHE_TYPE"
+        if [[ ! -z $KES_STATEDB_REDIS_ENDPOINT ]] && [[ $KES_STATEDB_REDIS_ENDPOINT != "" ]]; then
+            OPTIONS="$OPTIONS --statedb.cache.redis.endpoints $KES_STATEDB_REDIS_ENDPOINT"
+        fi
+
+        if [[ ! -z $KES_STATEDB_REDIS_PUBLISH ]] && [[ $KES_STATEDB_REDIS_PUBLISH -eq 1 ]]; then
+            OPTIONS="$OPTIONS --statedb.cache.redis.publish"
+        fi
+
+        if [[ ! -z $KES_STATEDB_REDIS_SUBSCRIBE ]] && [[ $KES_STATEDB_REDIS_SUBSCRIBE -eq 1 ]]; then
+            OPTIONS="$OPTIONS --statedb.cache.redis.subscribe"
+        fi
+    fi
+
+    if [[ ! -z $VM_INTERNALTX ]] && [[ $VM_INTERNALTX -eq 1 ]]; then
+        OPTIONS="$OPTIONS --vm.internaltx"
+    fi
+
+    if [[ ! -z $CHAINDATAFETCHER ]] && [[ $CHAINDATAFETCHER -eq 1 ]]; then
+        OPTIONS="$OPTIONS --chaindatafetcher"
+        if [[ ! -z $CHAINDATAFETCHER_MODE ]] && [[ $CHAINDATAFETCHER_MODE != "" ]]; then
+            OPTIONS="$OPTIONS --chaindatafetcher.mode $CHAINDATAFETCHER_MODE"
+        fi
+
+        if [[ ! -z $CHAINDATAFETCHER_KAFKA_BROKER ]] && [[ $CHAINDATAFETCHER_KAFKA_BROKER != "" ]]; then
+            OPTIONS="$OPTIONS --chaindatafetcher.kafka.brokers $CHAINDATAFETCHER_KAFKA_BROKER"
```

Comment:
This option can be set multiple times such as
`--chaindatafetcher.kafka.brokers [kafka-addr1] --chaindatafetcher.kafka.brokers [kafka-addr2] ... `. Please consider of this.